tests/test_basic.py

```diff
@@ -4,220 +4,192 @@
+    assert "[None]" in ret.stdout,  "time value is printed to console"
 
 def test_kml_time_invalid(script_runner):
-    ret = script_runner.run('python3', 'geoextent',
+    ret = script_runner.run('python', 'geoextent',
         '--path', 'Testdata/aasee.kml', 
         '--detail', 'bbox',
         '--time')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == 'invalid time format', "stderr should not be empty"
 
-def test_tif_bbox(script_runner):
-    ret = script_runner.run('python3', 'geoextent',
+def test_geotiff_bbox(script_runner):
+    ret = script_runner.run('python', 'geoextent',
         '--path', 'Testdata/wf_100m_klas.tif', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+    assert "[5.9153007564753155, 50.31025197410836, 9.468398712484145, 52.5307755328733]" in ret.stdout, "bbox is printed to console"
 
 def test_gpkg_bbox(script_runner):
-    ret = script_runner.run('python3', 'geoextent',
+    ret = script_runner.run('python', 'geoextent',
         '--path', 'Testdata/census2016_cca_qld_short.gpkg', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+    assert "[-43.7405, 96.8169, -9.14218, 167.998]" in ret.stdout, "bbox is printed to console"
 
 def test_csv_bbox(script_runner, tmpdir):
-    ret = script_runner.run('python3', 'geoextent',
-        '--path', 'Testdata/cities_NL.csv', 
+    ret = script_runner.run('python', 'geoextent',
+        '--path', 'Testdata/folder/cities_NL.csv', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[[10,20],[11,50]" in ret.stdout, "bbox is printed to console"
+    assert "[6.574722, 51.434444, 4.3175, 53.217222]" in ret.stdout, "bbox is printed to console"
 
 def test_csv_time(script_runner, tmpdir):
-    ret = script_runner.run('python3', 'geoextent',
-        '--path', 'Testdata/cities_NL.csv', 
+    ret = script_runner.run('python', 'geoextent',
+        '--path', 'Testdata/folder/cities_NL.csv', 
         '--detail', 'bbox',
         '--time')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "temporal extent is from 2017 ==> 2018" in ret.stdout, "time value is printed to console"
+    assert "[2018-09-30]" in ret.stdout, "time value is printed to console"
```

Comment:
Just repeating the above: for _all_ file formats that support time, have one test file with a time stamp, and one with a time period. For some, also have a test file with invalid dates, and a test file without dates.